src/Workspaces/CSharp/Portable/Classification/Worker_Preprocesser.cs

```diff
@@ -324,6 +328,29 @@ private void ClassifyLoadDirective(LoadDirectiveTriviaSyntax node)
         ClassifyDirectiveTrivia(node);
     }
 
+    private void ClassifyIgnoredDirective(IgnoredDirectiveTriviaSyntax node)
+    {
+        AddClassification(node.HashToken, ClassificationTypeNames.PreprocessorKeyword);
+        AddClassification(node.ColonToken, ClassificationTypeNames.PreprocessorKeyword);
+
+        // The first part (separated by whitespace) of content is a "keyword", e.g., 'sdk' in '#:sdk Test'.
+        // We only recognize some whitespace characters here for simplicity and performance.
+        if (node.Content.Text.IndexOfAny([' ', '\t']) is > 0 and var firstSpaceIndex)
+        {
+            var keywordSpan = new TextSpan(node.Content.SpanStart, firstSpaceIndex);
+            var stringLiteralSpan = TextSpan.FromBounds(node.Content.SpanStart + firstSpaceIndex, node.Content.FullSpan.End);
```

Comment:
Was there a reason we're classifying the whitespace character?

Reply:
Primarily simplicity. We didn't see any harm in it.  The entire line is free form anyways. So there might be many spaces. Easiest to just classify these chunks. 


Thoughts?